plugin/completion.py

```diff
@@ -98,9 +98,10 @@ def run(self, edit: sublime.Edit, item: Any, session_name: Optional[str] = None)
 
 class LspCompleteTextEditCommand(LspCompleteCommand):
 
-    def run(self, edit: sublime.Edit, item: Any, session_name: Optional[str] = None) -> None:
-        text_edit = item["textEdit"]
-        new_text = text_edit['newText']
+    def run(self, edit: sublime.Edit, item: CompletionItem, session_name: Optional[str] = None) -> None:
+        # LspCompleteTextEditCommand command will only be run if "textEdit" exist on the completion item
+        text_edit = cast(TextEdit, item.get("textEdit"))
+        new_text = text_edit["newText"]
```

Comment:
Would it not make sense to, instead of casting, add a null-check like:
```py
        if not text_edit:
            return
```
?
Even if currently all callers ensure that `textEdit` is set, it could save us in the feature if we do the check. And a cast always looks like a dirty hack.

Reply:
Yeah, I agree with you.